src/test/kotlin/ai/blindspot/ktoolz/extensions/OtherExtensionsTest.kt

```diff
@@ -151,4 +151,30 @@ internal class OtherExtensionsTest {
         }, block = { set(0, 0) }))
         assertEquals(1, listUnderTest[0])
     }
+
+    @Test
+    fun restrictLengthWithEllipsis() {
+        val result = "ABCDEFGHIJK".restrictLengthWithEllipsis(8, "elip")
+        assertEquals("ABCDelip", result)
+    }
+
+    @Test
+    fun toLongDebugString() {
+        assertEquals("Double(42.0)", 42.0.toLongDebugString())
+        assertEquals("MyClass[short]", "Any object".toLongDebugString("short","[]", className = "MyClass"))
```

Comment:
Please add a test that will use default values.

Reply:
`42.0.toLongDebugString()` uses default values (all three) - please explain what you mean.

Reply:
I didn't notice the first test, sorry.